css/arrow.css

```
@@ -86,3 +86,15 @@
 .flicking-arrow-disabled::after {
   background-color: #e6e6e6;
 }
+
+.flicking-arrow-prev.thin::before,
+.flicking-arrow-prev.thin::after,
+.flicking-arrow-next.thin::before,
+.flicking-arrow-next.thin::after{
+  height: 3px;
+}
+
+.flicking-arrow-prev.thin::after,
+.flicking-arrow-next.thin::after{
```

As other modifier classes are using names like `is-xxx`, this should be also changed like that.

```suggestion
.flicking-arrow-prev.is-thin::before,
.flicking-arrow-prev.is-thin::after,
.flicking-arrow-next.is-thin::before,
.flicking-arrow-next.is-thin::after{
  height: 3px;
}

.flicking-arrow-prev.is-thin::after,
.flicking-arrow-next.is-thin::after{
```